CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Splunk Enterprise SDK for JavaScript Changelog
 
+## v1.10.0
+
+### Minor changes
+
+* Added support for Splunk Enterprise 8.2
+* Added support for Node v14.
+* Dropped support for deprecated request http client.
+* Added support for [needle](https://www.npmjs.com/package/needle) http client library.
```

Comment:
Actually I think these two warrant moving into a new ```### Major changes``` section even though they are not breaking changes:
```
### Major changes
* Dropped support for deprecated request http client.
* Added support for [needle](https://www.npmjs.com/package/needle) http client library.
* ```

Reply:
Changes done